ecl/hqlcpp/hqliproj.cpp

```diff
@@ -1079,7 +1079,10 @@ IHqlExpression * ComplexImplicitProjectInfo::createOutputProject(IHqlExpression
     OwnedHqlExpr seq = createSelectorSequence();
     OwnedHqlExpr left = createSelector(no_left, ds, seq);
     OwnedHqlExpr self = getSelf(queryOutputRecord());
-    IHqlExpression * transform = createMappingTransform(self, left);
+    MultiErrorReceiver errs;
+    ECLlocation dummyLocation(0, 0, 0, NULL);  // MORE - shame there is not a global one that could be used for this?
```

Comment:
Could use the default constructor, or pass ds as the argument.  Hopefully it will never use the argument, so doesn't really matter.